Support/Multitenancy/Multiple-Ironic-conductors/generate_unique_nodes.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+rm -rf macaddrs uuids nodes.json node.json
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+function macgen {
+  hexdump -n 6 -ve '1/1 "%.2x "' /dev/random | awk -v a="2,6,a,e" -v r="$RANDOM" 'BEGIN{srand(r);}NR==1{split(a,b,",");r=int(rand()*4+1);printf "%s%s:%s:%s:%s:%s:%s\n",substr($1,0,1),b[r],$2,$3,$4,$5,$6}'
+}
+
+function generate_unique {
+  func=$1
+  store_file=$2
+  newgen=$($func)
+  if [[ ! -f "$store_file" || $(grep "$newgen" "$store_file") == "" ]]; then
+    echo "$newgen" >>"$store_file"
+    echo "$newgen"
+    return
+  fi
+  $func
```

Comment:
```suggestion
  "${func}"
```
Unless func contains spaces.